nailgun/entities.py

```diff
@@ -643,7 +643,7 @@ class Meta(object):
         server_modes = ('sat')
 
 
-class ContentViewVersion(Entity, EntityReadMixin):
+class ContentViewVersion(Entity, EntityReadMixin, EntityDeleteMixin):
```

Comment:
This is good and makes sense. It allows us to make use of `DELETE /katello/api/content_view_versions/:id`.